src/policy/policy.h

```diff
@@ -19,6 +19,8 @@ static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 750000;
 static const unsigned int DEFAULT_BLOCK_MIN_SIZE = 0;
 /** Default for -blockprioritysize, maximum space for zero/low-fee transactions **/
 static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 0;
+/** Default for -blockmaxcost, which control the range of block costs the mining code will create **/
+static const unsigned int DEFAULT_BLOCK_MAX_COST = 3000000;
```

Comment:
I suggest we change this to 4000000 so that pre-activation, miners who are setting only `-blockmaxsize` don't see a potential behavior change.

FYI this is what is currently breaking `qa/rpc-tests/pruning.py`.